### ISSUE TITLE
Doesn't export all items

### ISSUE THREAD

**Issue by pooria121** [boussadjra/vueye-table#16]
We are using VueyeComponent to display a very paginated large table (with 300,000 rows). 
Currently, the excel export button would export rows that are currently displayed:
```
const items = computed(() => {
            return props.sharedState.currentPageItems;
        });
```
However, many applications (including ours) require exporting the entire table for further processing. 
Displaying the entire 300,000 in a single page can get us by, but most often result in crashing the HTML page. 

I would argue that the Excel export button should export all the data passed to the table, or at least there should be a flag to set/control this behavior.

**Comment by boussadjra** [comment]
Exporting a big amount of data could also crashe the app since the exporting process creates a table under hood then export it to excel

**Comment by pooria121** [comment]
Agreed, but it hard to define "big". For example, in our case, 300000 is not big enough to crash the app when you create a table in memory that is not going to be displayed  (and just exported!) - but forcing 300000 to be displayed before being exported (current code) would definitely crash due to unnecessary rendering. Thus, I think it is a good idea to give developers the agility to configure this behavior.

**Comment by boussadjra** [comment]
I didn't have that amount of data to test it, you could take this [code](https://github.com/boussadjra/vueye-table/blob/b5d49ed6e8897786c1ddc56b72d7090fd452f961/src/components/subcomponents/VeHeader.vue#L88) and try it out and let me know if that doesn't affect the performance
